python/AmcCarrierCore/AppTop/_AppTop.py

```diff
@@ -53,6 +54,14 @@ def __init__(   self,
             value        = False,
         ))
 
+        self.add(pr.LocalVariable(
+            name         = 'JesdHealthStatus',
+            mode         = 'RO',
+            localGet     = lambda: self.JesdHealth(),
+            value        = False,
+            pollInterval = 5,
+        ))
+
```

Comment:
@ruck314 Can we set this variable's `pollInterval` to `0` by default? For SMuRF we already have a function to check this status which is called from the high level application.

Reply:
1f6464aca9860ace092354040b5d6c6fbb307596
Done